shardingsphere-features/shardingsphere-encrypt/shardingsphere-encrypt-core/src/main/java/org/apache/shardingsphere/encrypt/rewrite/token/generator/impl/EncryptCreateTableTokenGenerator.java

```diff
@@ -73,23 +75,66 @@ protected boolean isGenerateSQLTokenForEncrypt(final SQLStatementContext sqlStat
         return result;
     }
     
-    private SubstitutableColumnNameToken getCipherColumnToken(final String tableName, final String columnName, final ColumnDefinitionSegment column, final int stopIndex) {
-        return new SubstitutableColumnNameToken(stopIndex + 1, column.getColumnName().getStopIndex(), 
+    private SQLToken getCipherColumnToken(final String tableName, final String columnName, final ColumnDefinitionSegment column, final int stopIndex) {
```

Comment:
@tuichenchuxin Why change return type to SQLToken?

Reply:
@strongduanmu Because when config dataType then will return `EncryptConfigDataTypeToken` , not `SubstitutableColumnNameToken` again.